.github/workflows/windows-npm.yml

```diff
@@ -0,0 +1,64 @@
+name: 'Tests on Windows: `nvm install`'
+
+on: [pull_request, push]
+
+jobs:
+  node:
+    name: 'MSYS fail prefix nvm install'
+    runs-on: windows-latest
+    steps:
+      - name: Retrieve nvm
+        shell: bash
+        run: |
+          mkdir vp "$HOME/.nvm"
+          curl -sSLo "$HOME/.nvm/nvm.sh" "https://raw.githubusercontent.com/${GITHUB_REPOSITORY}/${GITHUB_SHA}/nvm.sh"
+          source "$HOME/.nvm/nvm.sh"
+          ! nvm install --lts
+
+  nodes:
+    name: 'MSYS nvm install'
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        npm-node-version:
+          - '--lts'
+          - '--default 12'
+          - '--no-progress 10'
+
+    steps:
+      - name: Retrieve nvm
+        shell: bash
+        run: |
+          mkdir vp "$HOME/.nvm"
+          curl -sSLo "$HOME/.nvm/nvm.sh" "https://raw.githubusercontent.com/${GITHUB_REPOSITORY}/${GITHUB_SHA}/nvm.sh"
+          source "$HOME/.nvm/nvm.sh"
+          unset npm_config_prefix
+          nvm install ${{ matrix.npm-node-version }}
+
+  wsl_nodes:
+    name: 'WSL nvm install'
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        wsl-distrib:
+          - Debian
+          - Alpine
+          - Ubuntu-18.04
+        npm-node-version:
+          - '--lts'
+          - '11'
+    steps:
+      - uses: Vampire/setup-wsl@v1
+        with:
+          distribution: ${{ matrix.wsl-distrib }}
+          additional-packages: bash curl ca-certificates
+      - name: Retrieve nvm on WSL
+        shell: wsl-bash {0}
+        env:
+          GITHUB_REPOSITORY: ${{ github.repository }}
+          GITHUB_SHA: github.sha
+        run: |
+          mkdir -p "$HOME/.nvm"
+          curl -sSLo "$HOME/.nvm/nvm.sh" "https://raw.githubusercontent.com/${{ github.repository }}/${{ github.sha }}/nvm.sh"
```

Comment:
Indeed the install script point to the main repo so it would not take my changes, so obviously it would fail yes.
I can restore the env var, I kept it anyway on a separate branch to be able to use it on my side even if not integrated in the main repo yet. I would still have to set those env pointing to  the repo/sha so it really would check everything using the current pull request.